test/common/common.go

```diff
@@ -212,3 +216,31 @@ func OutputDebugInfo(testName string, kubeconfig string, additionalResources ...
 		_, _ = utils.KubectlWithOutput("get", resource, "--all-namespaces", "-o", "yaml", "--kubeconfig="+kubeconfig)
 	}
 }
+
+// IsMinimumVersion detects OCP versions given an x.y version lower bound
+func IsMinimumVersion(minVersion string) bool {
+	clusterVersion, err := ClientManagedDynamic.Resource(GvrClusterVersion).Get(
+		context.TODO(),
+		"version",
+		metav1.GetOptions{},
+	)
+
+	if err != nil && k8serrors.IsNotFound(err) {
+		// no version CR, not ocp
+		klog.V(5).Infof("This is not an OCP cluster")
+
+		return false
+	}
+
+	desired := clusterVersion.Object["status"].(map[string]interface{})["desired"]
+	version, _ := desired.(map[string]interface{})["version"].(string)
+
+	klog.V(5).Infof(fmt.Sprintf("OCP Version " + version))
+
+	// Convert to valid semantic versions by adding the "v" prefix
+	minSemVer := fmt.Sprintf("v%s", minVersion)
+	ocpSemVer := semver.MajorMinor(fmt.Sprintf("v%s", version))
+
+	// Compare returns: 0 if ver1 == ver2, -1 if ver1 < ver2, or +1 if ver1 > ver2
+	return semver.Compare(ocpSemVer, minSemVer) != -1
```

Comment:
Nit: it might be more usual to write this as `>=0`. In other places I've seen with similar `Compare` functions, the value isn't always restricted to `[-1, 0, 1]`. But it's also possible that `!= -1` is preferred by this library, so in that case just let me know!

Reply:
In case the library changes, that does sound more resilient. I'll make the update!